sdk/java/src/main/java/com/chain/api/CoreConfig.java

```diff
@@ -0,0 +1,141 @@
+package com.chain.api;
+
+import com.chain.http.*;
+import com.chain.exception.*;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.*;
+
+public class CoreConfig {
+
+  public static class Info {
+    public String state;
+
+    @SerializedName("is_configured")
+    public boolean isConfigured;
+
+    @SerializedName("configured_at")
+    public Date configuredAt;
+
+    @SerializedName("is_signer")
+    public boolean isSigner;
+
+    @SerializedName("is_generator")
+    public boolean isGenerator;
+
+    @SerializedName("generator_url")
+    public String generatorUrl;
+
+    @SerializedName("generator_access_token")
+    public String generatorAccessToken;
+
+    @SerializedName("blockchain_id")
+    public String blockchainId;
+
+    @SerializedName("block_height")
+    public long blockHeight;
+
+    @SerializedName("generator_block_height")
+    public long generatorBlockHeight;
+
+    @SerializedName("generator_block_height_fetched_at")
+    public Date generatorBlockHeightFetchedAt;
+
+    @SerializedName("network_rpc_version")
+    public int networkRpcVersion;
+
+    @SerializedName("core_id")
+    public String coreId;
+
+    public String version;
+
+    @SerializedName("build_commit")
+    public String buildCommit;
+
+    @SerializedName("build_date")
+    public String buildDate;
+
+    @SerializedName("build_config")
+    public BuildConfig buildConfig;
+
+    public Map<String, Object> health;
```

Comment:
is there any more schema to this that we can provide? it's a little unclear how to use this as-is. If I understand the cored code right, this map would look something like:
```
{
  "errors": {
    "generator": "error generating new block"
  }
}
```

Maybe instead of a Map<String,Object> this should be a separate class? Even if we only expose a method that returns this map, it'll give us the flexibility to introduce more convenience methods down the road.